build.conf

```diff
@@ -6,50 +6,47 @@ BUILD_DIR="build"
 # This is also defined independently in "publish_dist.sh".
 DIST_DIR="dist"
 
+# Setting this as 0 disables tests during building (skips chevahbs_test phase).
+PYTHIA_BUILD_TESTS="${PYTHIA_BUILD_TESTS:-1}"
+
 # Python and lib versions.
-PYTHON_BUILD_VERSION="3.11.7"
-LIBFFI_VERSION="3.4.4"
-ZLIB_VERSION="1.3"
+PYTHON_BUILD_VERSION="3.11.9"
+LIBFFI_VERSION="3.4.6"
+ZLIB_VERSION="1.3.1"
 BZIP2_VERSION="1.0.8"
 # To check the signature of the XZ download:
 # "gpg --keyserver-options auto-key-retrieve --verify xz-*.sig xz-*.gz".
-XZ_VERSION="5.4.5"
+XZ_VERSION="5.6.2"
 # Statically build the BSD libedit on selected platforms to get the
 # readline module available without linking to the GPL-only readline libs.
 # If there's a need to reenable this, our libedit patch for Python 3.9 was
 # https://github.com/chevah/pythia/pull/5/commits/09c128154d23feb6b1a7cb5a8d79.
 # A newer patch is available at https://github.com/python/cpython/issues/57710.
 LIBEDIT_VERSION="20170329-3.1"
 # Our OpenSSL libs are only used for Python's "ssl" module lately.
-OPENSSL_VERSION="3.1.4"
+OPENSSL_VERSION="3.0.14"
 # Use the version of the "sqlite-autoconf-VERSION.tar.gz" upstream download.
 # To get its SHA3-256 signature: "openssl dgst -sha3-256 sqlite-autoconf-*".
 # When updating this, also update the year in src/sqlite/chevahbs, if needed.
-SQLITE_VERSION="3440200"
+SQLITE_VERSION="3460000"
 
 # If latest get-pip.py stops working on older platforms, use something like:
 # https://web.archive.org/web/20220122185015/https://bootstrap.pypa.io/.
 BOOTSTRAP_GET_PIP="https://bootstrap.pypa.io/get-pip.py"
```

Comment:
With latest Python,  I don't think that this is needed.

You should be able to just do `python -m ensurepip --upgrade`

Reply:
Fixed through a3c646f.

Reply:
Unfortunately, this is not a solution on Windows, where the upstream package we use doesn't seem to include `ensurepip`:
```
## Bootstrapping pip... ##
D:\a\pythia\pythia\build\python3.11-windows-x64\lib\python.exe: No module named ensurepip
```

Reply:
Thanks. Not sure why ensurepip is not available on Windows :(

Reply:
The embeddable package on Windows is cut to the bone.

Went back to using `get-pip.py`, but only on Windows: https://github.com/chevah/pythia/pull/59/files/cbd3bc16b1cbb257d59f0e2cf5dedff1ca7ef039..4d6db81c9bd9df5161200c5ae8b70ac6f5bdd107